### PR TITLE
Add info about LXD bug

### LINKED ISSUE #539: When deploying local kubernetes cluster, kurbernetes-worker nodes miss ephemeral storage
Hi,

I'm trying to get started with Charmed Kubernetes and followed your local installation guide from here;

https://ubuntu.com/kubernetes/docs/install-local

The kubernetes-master node is stuck in waiting for "Waiting for 3 kube-system pods to start".
And that's to be expected because the kubernetes-worker nodes are NotReady.

When I use describe on one of the nodes, this message shows up constantly;

Warning  InvalidDiskCapacity      12s    kubelet     invalid capacity 0 on image filesystem

When I ssh into that worker and run a disk check;

Filesystem                         Size  Used Avail Use% Mounted on
/dev/mapper/ubuntu--vg-ubuntu--lv   62G   31G   28G  53% /
none                               492K  4.0K  488K   1% /dev
udev                               1.9G     0  1.9G   0% /dev/fuse
tmpfs                              100K     0  100K   0% /dev/lxd
tmpfs                              100K     0  100K   0% /dev/.lxd-mounts
tmpfs                              2.0G     0  2.0G   0% /dev/shm
tmpfs                              394M  1.1M  393M   1% /run
tmpfs                              5.0M     0  5.0M   0% /run/lock
tmpfs                              2.0G     0  2.0G   0% /sys/fs/cgroup
tmpfs                              1.0M     0  1.0M   0% /var/snap/lxd/common/ns
tmpfs                              394M     0  394M   0% /run/user/1000

So, what am I doing wrong?

### PR DESCRIPTION
**Pull request by evilnick**
Fixes #539

### CODE REVIEW

File: pages/k8s/install-local.md [hunk 1]
```diff
@@ -135,11 +135,24 @@ All that remains is to deploy **Charmed Kubernetes**. A simple install can be ac
...
 This will install the latest stable version of **Charmed Kubernetes** with
 the default components and configuration. If you wish to customise this install
 (which may be helpful if you are close to the system requirements), please see
 the [main install page][install].
 
+<div class="p-notification--warning"><p markdown="1" class="p-notification__response">
+<span class="p-notification__status">Bug Warning:</span>
+There is currently a bug, <a href="https://bugs.launchpad.net/charm-kubernetes-worker/+bug/1903566"> LP#1903566</a>,
+which prevents <code>Kubelet</code> from running properly on LXD. Until this is fixed, a 
+workaround is to configure kubelet to protect the kernel defaults:
```
Comment (Cynerva) [inline pages/k8s/install-local.md:148]: Other way around, setting this disables kernel default protection. I would rewrite this to say something like:

```
Until this is fixed, a workaround is to configure kubelet to disable kernel default protection:
```

or

```
Until this is fixed, a workaround is to configure kubelet to override kernel defaults:
```

File: pages/k8s/install-local.md [hunk 2]
```diff
@@ -135,11 +135,24 @@ All that remains is to deploy **Charmed Kubernetes**. A simple install can be ac
...
+  
+<pre><code>
+juju config kubernetes-worker kubelet-extra-config='{protectKernelDefaults: false}'
+</code></pre>
```
Comment (Cynerva) [inline pages/k8s/install-local.md:152]: The `<pre>` and `</pre>` seem to be showing up as text on the preview site. Not sure what's going on there.

https://deploy-preview-557--cdk-docs-next.netlify.app/kubernetes/docs/install-local

**Review by Cynerva — CHANGES_REQUESTED** [review]

File: pages/k8s/install-local.md (hunk 1 above)
Comment (evilnick) [reply pages/k8s/install-local.md:148]: ah, okay. thanks!

File: pages/k8s/install-local.md (hunk 2 above)
Comment (evilnick) [reply pages/k8s/install-local.md:152]: yeah, I know, trying to fix that

Comment (evilnick) [reply pages/k8s/install-local.md]: Screenshot from 2021-04-23 16-18-36

no idea why `<pre>`  doesn't work, but this looks right now at least

**Review by Cynerva — APPROVED** [review]

**Pull request merged**